.github/workflows/ci.yml

```
@@ -0,0 +1,54 @@
+name: Ci
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-[a-zA-Z]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-[a-zA-Z]+.[1-9]'
```

Todo esto se puede resumir en una sola línea (8-10):

```suggestion
      - 'v*'
```